setup.py

```diff
@@ -26,6 +26,7 @@
     author_email = 'mans.magnusson@scilifelab.se',
     url = 'http://github.com/moonso/genmod',
     license = 'MIT License',
+    python_requires="~=3.8.0",
```

Comment:
```suggestion
    python_requires="~=3.8",
```
We may want to loosen the requirement on the patch version. However, I'm no python expert so I'll leave the decision to you :)

Reply:
@jemten This relaxes the python requirement to  `v3.8.*`.
I'm basing this on https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release


> The padding rules for release segment comparisons means that the assumed degree of forward compatibility in a compatible release clause can be controlled by appending additional zeros to the version specifier:
```
~= 2.2.0
are equivalent to:
>= 2.2.0, == 2.2.*
```

as well as my testing locally.